platform/source/mbed_retarget.cpp

```diff
@@ -516,8 +608,13 @@ extern "C" FILEHANDLE PREFIX(_open)(const char *name, int openflags)
     }
 #endif
     return open(name, openflags_to_posix(openflags));
+#else
+    // Everything goes to the same output
+    return 1;
+#endif // !(MBED_CONF_PLATFORM_STDIO_MINIMAL_CONSOLE_ONLY)
 }
 
+#if !(MBED_CONF_PLATFORM_STDIO_MINIMAL_CONSOLE_ONLY)
```

Comment:
Remove also the declaration in mbed_retarget.h

Reply:
[Done](https://github.com/hugueskamba/mbed-os/compare/9ab95d568e3b1b18b96f057a9ddf6a43bbeaada9..984d3d625c22b050e42c283372303827d347f0d4).